src/Features/LanguageServer/Protocol/Handler/Initialize/InitializeHandler.cs

```diff
@@ -54,6 +54,7 @@ public Task<LSP.InitializeResult> HandleRequestAsync(LSP.InitializeParams reques
                     DocumentFormattingProvider = true,
                     DocumentRangeFormattingProvider = true,
                     DocumentOnTypeFormattingProvider = new LSP.DocumentOnTypeFormattingOptions { FirstTriggerCharacter = "}", MoreTriggerCharacter = new[] { ";", "\n" } },
+                    OnAutoInsertProvider = new LSP.DocumentOnAutoInsertOptions { TriggerCharacters = new[] { "'", "/", "\n" } },
```

Comment:
Having both comment chars is going to cause more method calls over the write, but it doesn't look like there is a way to make this smart and separate out trigger characters by language, since we do the same thing for completion characters.

@dibarbet do we need an issue to fix this? I couldn't find one.

Reply:
I think https://github.com/dotnet/roslyn/issues/46195 may take care of this - then we can export different handlers for C# and VB with different characters